grabbit/src/main/groovy/com/twcable/grabbit/client/batch/ClientBatchJob.groovy

```diff
@@ -42,8 +42,9 @@ class ClientBatchJob {
     public static final String WORKFLOW_CONFIGS = "workflowConfigIds"
```

Comment:
Maybe. Though I still like the way we build out the Client (& Server) batch job. I think because there are so many things to configure, it may still make sense.